www/layouts/partials/home_course_cards.html

```diff
@@ -1,7 +1,8 @@
 <div class="new-courses course-cards standard-width container mx-auto mt-3">
 {{ $studioBaseUrl := getenv "OCW_STUDIO_BASE_URL" }}
-{{ if $studioBaseUrl }}
-{{  with getJSON (print (strings.TrimSuffix "/" $studioBaseUrl) "/api/websites/?type=course")  }}
+{{ $courseStarterSlug := getenv "OCW_IMPORT_STARTER_SLUG" }}
```

Comment:
Are the environment variables defined for ocw-studio the same environment variables defined for ocw-hugo-themes?

Reply:
Good point, there will need to be a `salt-ops` PR to set these since the current OCW Next site build server is unique.  Either way, I think we should keep the name consistent.

Reply:
Can we just pick a constant and use that everywhere? I'm not sure why this was made an environment variable

Reply:
When we started development of `ocw-studio`, a "course" starter was added directly to the repo in the `localdev` folder, but it ended up being used for more than just "local development."  It's used as a fixture essentially in every deployed instance of `ocw-studio` so even in RC and Prod we end up with a `course` starter already existing.  The starter in `ocw-hugo-projects` was named `ocw-course` as to not conflict.  They can't be named the same thing, so it was made configurable so you can use either the local version checked into `ocw-studio` or the imported version from `ocw-hugo-projects`.  I made a PR to set the value on the OCW Next build server here: https://github.com/mitodl/salt-ops/pull/1456